topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md

```diff
@@ -210,21 +211,22 @@ What we referenced here as a *sampleMetadata* file corresponds to a table contai
 
 A sample metadata file contains various information for each of your raw files:
 - **Classes** which will be used during the preprocessing steps
-- **Number of batches** which will be useful for a batch correction step, along with sample types (pool/sample) and injection order
-- Different **experimental conditions** which can be used for the statistics
+- **Analytical batches** which will be useful for a batch correction step, along with **sample types** (pool/sample) and **injection order**
+- Different **experimental conditions** which can be used for statistics
 - Any information about samples that you want to keep, in a *column* format
 
 The content of your sample metadata file has to be filled by you, since it is not contained in your raw data.
 Note that you can either:
 - Upload an existing metadata file
 - Use a template to create one (because it can be painful to get the sample list without misspelling or omission)
-  1. Generate a template with the tool **xcms get a sampleMetadata file** {% icon tool %}
+  1. Generate a template with the **xcms get a sampleMetadata file** {% icon tool %} tool
   2. Fill it using your favorite table editor (Excel, LibreOffice)
   3. Upload it within Galaxy
 
 > ### {% icon tip %} Optional: Generate the right template with **xcms get a sampleMetadata file** {% icon tool %}
```

Comment:
I found that there was quite a lot of helpful information about the metadata file, but it is not immediately needed when following the tutorial so I was not sure what I can skip and what not. I was wondering if all the detailed information could go into a hidden section that is called "generating your own metadata file" or similar.

Reply:
Thank you for your comment!

Indeed it is not mandatory since we provide a metadata file with everything all right in the tutorial. Nonetheless, this step is very important in the process. Through years we saw how this sampleMetadata question could easily turn into trouble for users. This is why we had two concerns:
- not providing an "alright" file to users would be a problem for automonous learning in various way
- displaying this part as something kind of hidden that could be skipped without remorse would be a mistake

This is why we chose this format here that is kind of compromise.